app/views/extractorMetrics.scala.html

```diff
@@ -0,0 +1,48 @@
+@(extractorName: String, average: String, lastweeksubmitted: Int, lastmonthsubmitted: Int)(implicit user: Option[models.User])
+@import api.Permission
+
+@main("Extractor Details") {
+    <script src="@routes.Assets.at("javascripts/deleteUtils.js")" type="text/javascript"></script>
+    <script src="@routes.Assets.at("javascripts/extractors/extractors.js")" type="text/javascript"></script>
+    <style>
+            strong        { text-align: right; }
+            fieldset      { margin-top: 5px; }
+            .form-actions { margin-top: 15px; }
+    </style>
+
+    <ol class="breadcrumb">
+        <li><span class="glyphicon glyphicon-fullscreen"></span> <a href="@routes.Extractors.selectExtractors()">Extractor Catalog</a></li>
+        <li><span class="glyphicon glyphicon-search"></span> @extractorName</li>
+    </ol>
+
+    <div class="page-header"><h1>@extractorName Metrics Details</h1></div>
+
+    <div class="row">
+        <div class="col-sm-8">
+            <form>
+                <fieldset>
+
+                    <div class="row form-group">
+                        <strong class="col-sm-3" for="extractorNameField">Last 24 Hours Execution Average</strong>
```

Comment:
If we make these `col-sm-4` and `col-sm8` the default text will not wrap around:

![Screen Shot 2020-09-29 at 1 20 36 PM](https://user-images.githubusercontent.com/599725/94599855-adb9a580-0256-11eb-95a4-61eeb5389daf.png)